src/styles/executive.css

```diff
@@ -0,0 +1,24 @@
+body
+{
+    background-color: black;
+    color: white;
+    text-align: center;
```

Comment:
This *may* be a booger when we try to style consistently across branches. I wouldn't remove it now or anything, but if we have any alignment discrepancies with your branch, check this out first.